src/bench/addrman.cpp

```diff
@@ -126,8 +126,10 @@ static void AddrManSelectByNetwork(benchmark::Bench& bench)
 
     FillAddrMan(addrman);
 
+    std::vector<Network> net_i2p = {NET_I2P};
+
     bench.run([&] {
-        (void)addrman.Select(/*new_only=*/false, NET_I2P);
+        (void)addrman.Select(/*new_only=*/false, net_i2p);
```

Comment:
This should work as well (untested):

```cpp
(void)addrman.Select(/*new_only=*/false, {NET_I2P});
```

Same in `src/test/addrman_tests.cpp`.

Reply:
It doesn't work with vector I guess. But since I'll change it to `unordered_set`, it will work.

Reply:
Does not matter now, but it should work with vector as well.